main/src/modules/Jvm.scala

```diff
@@ -267,19 +262,12 @@ object Jvm {
   def createAssembly(inputPaths: Agg[os.Path],
                      manifest: JarManifest = JarManifest.Default,
                      prependShellScript: String = "",
-                     base: Option[os.Path] = None,
```

Comment:
I had to remove this because we need to process both the project's classes and dependencies with the same shader instance.